src/main/java/frc/robot/subsystems/CoralRoller.java

```diff
@@ -143,6 +147,11 @@ public void outtake() {
     outtakeTimer.restart();
   }
 
+  public void outtake(double outtakeSpeed) {
+    setGoalVelocity(outtakeSpeed);
```

Comment:
Since this could take a positive or negative speed, I suggest renaming this:
```suggestion
  public void runRollers(double speed) {
      setGoalVelocity(speed);
```

Reply:
We no longer use `outtake`, so I replace this method and the `intake` method with calls to `setGoalVelocity` instead.